test/gui/shared/steps/steps.py

```diff
@@ -744,3 +744,56 @@ def step(context, resource, content):
 
     print("file has been overwritten")
     waitForFileToBeSynced(context, resource)
+
+
+def enableVFSSupport(vfsBtnText):
+    mouseClick(
+        waitForObjectItem(names.stack_folderList_QTreeView, "_1"),
+        718,
+        27,
```

Comment:
what is in (718,27)?

Reply:
VFS button is there. But I have changed it to (718, 27) so that every coordinates for VFS button are same.